build_web_compilers/lib/src/archive_extractor.dart

```diff
@@ -6,8 +6,6 @@ import 'dart:async';
 
 import 'package:archive/archive.dart';
 import 'package:build/build.dart';
-import 'package:build_runner/build_runner.dart'
```

Comment:
weird how was this not an analysis error?

Reply:
It's fine to have the same class imported through two files and the `build_runner` one still exports these.

I'll probably move this to a separate PR though as cleanup.